Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/common/ClusterManagerEx.java

```diff
@@ -69,6 +74,54 @@ public static ClusterManagerEx getInstance() {
         return instance;
     }
 
+    public String getClusterConnectionString(final @NotNull String clusterName) {
```

Comment:
I would like to ask you to add `@NotNull` or `@Nullable` for all new functions, please

Reply:
I agree with that. It's just format issue. Fixed. Thanks.